### PR TITLE
KIP-3: Remove KUMO token from Stability Pool

### PR DESCRIPTION
**Pull request by arthurka-o**
Removes any functionality related to the KUMO token in the Stability Pool contract and it's interface.
Also, removes the only way to issue the KUMO tokens.

### CODE REVIEW

**Review by marceljay — APPROVED** [review]
Great work, nicely structured PR!  🚀

Unfortunately the tests didn't run, and i'm trying to find out why. I will approve this anyway, since I assume the tests would fail no matter what. Too bad though we don't have job for compilation only.

**Comment by arthurka-o** [comment]
Oh, it didn't run because I was switching between draft and ready states specifically not to run.
Tests will fail anyway.

File: packages/contracts/contracts/StabilityPool.sol [hunk 1]
```diff
@@ -715,32 +715,6 @@ contract StabilityPool is KumoBase, CheckContract, IStabilityPool {
         return vars.SPAssetGain + vars.StakingAssetGain;
     }
 
-    function _getETHGainFromSnapshots(
```
[the remark this answers is not in the document]
Comment (arthurka-o) [reply packages/contracts/contracts/StabilityPool.sol:718]: As I remember. there was a two identical functions with a slight different names. And one of them was unused (this one).
Probably came from the Vesta.

File: packages/contracts/contracts/StabilityPool.sol [hunk 2]
```diff
@@ -234,8 +216,8 @@ contract StabilityPool is KumoBase, CheckContract, IStabilityPool {
     // Tracker for KUSD from fees held in the pool. Changes when users borrow/redeem, and when users withdraw gains.
     uint256 internal totalKUSDGains;
 
-    uint256 public F_ASSET; // Running sum of Asset fees per-KUMO-staked
-    uint256 public F_KUSD; // Running sum of KUMO fees per-KUMO-staked
+    uint256 public F_ASSET; // Running sum of Asset fees per-KUSD-staked
```
[the remark this answers is not in the document]
Comment (arthurka-o) [reply packages/contracts/contracts/StabilityPool.sol:219]: It was `F_ETH` before. But I didn't really think about it.

**Pull request merged**